tests/integration/test_kyuubi.py

```diff
@@ -31,6 +31,7 @@
 
 
 @only_on_microk8s
+@only_on_localhost  # TODO remove when kyuubi is fixed
```

Comment:
Temporarily disable the tests until https://github.com/canonical/kyuubi-k8s-operator/pull/70 is merged or the failures are otherwise fixed.